caproto/_dbr.py

```diff
@@ -40,6 +40,12 @@
 class_name_t = 40 * ctypes.c_char  # epicsOldString
 
 
+def epics_timestamp_to_unix(secondsSinceEpoch, nanoSeconds):
+    '''UNIX timestamp (seconds) from Epics TimeStamp structure'''
+    return (EPICS2UNIX_EPOCH + secondsSinceEpoch + 1.e-6 *
```

Comment:
Paging @hhslepicka, need assistance on aisle 7; committer is lacking an appropriate meme response